### ISSUE TITLE
keras.layers.normalization deprecated

### ISSUE THREAD

**Issue by sqing73** [the-deep-learners/deep-learning-illustrated#16]
`from keras.layers.normalization import BatchNormalization `
could be changed to
`from tensorflow.keras.layers import BatchNormalization`

**Comment by jonkrohn** [comment]
Yep, you're right! For the sake of consistency with the book, I'm going to leave it as-is in this particular repo, but you can see the updated (TensorFlow 2.x) code that uses the `tensorflow.keras.` nomenclature in the newer, analogous repo [here](https://github.com/jonkrohn/DLTFpT/).
